### DISCUSSION TITLE
FreeRTOS Call's

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#15462] [Q&A] [answered]
Any major side effects in Tasmota when using some FreeRTOS calls, like:
~~~
    vTaskDelay (200/portTICK_PERIOD_MS);
~~~

**Comment by barbudor** [comment, marked as answer]
Arduino's core thought about it first
https://github.com/espressif/arduino-esp32/blob/8dc70e0add06e75da5a255ba94fe86df98ae5ef1/cores/esp32/esp32-hal-misc.c#L144

**Comment by trlafleur** [comment]
Well that’s good news.   Make it easy to use.

Thanks



On Mon, Apr 25, 2022 at 2:46 PM Barbudor ***@***.***> wrote:

> Arduino's core thought about it first
>
> https://github.com/espressif/arduino-esp32/blob/8dc70e0add06e75da5a255ba94fe86df98ae5ef1/cores/esp32/esp32-hal-misc.c#L144
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/arendst/Tasmota/discussions/15462#discussioncomment-2634202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABC4EK232TCCSLUKH3ODNI3VG4HDVANCNFSM5UJ2N4IQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur
